parquet-hadoop/src/main/java/org/apache/parquet/hadoop/rewrite/ParquetRewriter.java

```diff
@@ -366,6 +366,10 @@ private void processChunk(ColumnChunkMetaData chunk,
 
     ColumnIndex columnIndex = reader.readColumnIndex(chunk);
     OffsetIndex offsetIndex = reader.readOffsetIndex(chunk);
+    BloomFilter bloomFilter = reader.readBloomFilter(chunk);
+    if (bloomFilter != null) {
+      writer.addBloomFilter(chunk.getPath().toDotString(), bloomFilter);
```

Comment:
If the column is masked to null, we should not keep the bloom filter. Similarly, if the column is encrypted, we should make sure the bloom filter is encrypted. Could you please add two tests to make sure they are handled?

Reply:
Sorry for the late response. Added the related UTs.